docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -0,0 +1,21 @@
+## How does transaction finality work?
```

Comment:
how it _works_ sounds a bit odd to me. maybe "when does a transaction become final?"